rmgpy/data/kinetics/kineticsTest.py

```diff
@@ -1091,3 +1094,37 @@ def test_add_atom_labels_for_reaction_3(self):
         self.assertIn('*2',found_labels)
         self.assertIn('*3',found_labels)
 
+
+###################################################
+class TestLoadFilterFits(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Load an RMG job to test loading the YAML file with custom filter criteria.
+        """
+        folder = os.path.join(os.path.dirname(__file__), '../../solver/files/listener/')
+        inputFile = os.path.join(folder, 'input.py')
+        chemkinFile = os.path.join(folder, 'chemkin/chem.inp')
+        spc_dict = os.path.join(folder, 'chemkin/species_dictionary.txt')
+
+        # set-up RMG object
+        cls.rmg = RMG()
+        cls.rmg = loadRMGPyJob(inputFile, chemkinFile, spc_dict, generateImages=False, checkDuplicates=False)
```

Comment:
It seems like this test could be simplified significantly since the RMG instance isn't actually necessary.

Reply:
Simplified, though not sure if in the most elegant way.